package.json

```diff
@@ -93,5 +93,8 @@
       "admin/",
       "packages/"
     ]
+  },
+  "dependencies": {
+    "winston": "^3.10.0"
```

Comment:
wrong installation location. you will have to do the following steps

```bash
npm uninstall winston  # from root
# your changes to both package and package-lock 
# should reset to what they were before
# OR
# you can do git reset HEAD~1, discard changes to these files,
# and run npm run reinstall to go back to the previous state
# and then 
cd ./api
npm i winston
```

Reply:
make sure to run `npm run reinstall` in either case to make sure package-lock isn't pointing to your cache.